src/environment.ts

```diff
@@ -6,24 +6,30 @@ import Constants from 'expo-constants';
  * the env var values. This can then be returned via the getEnvVars function.
  */
 const ENV = {
-    dev: {
-        apolloURI: 'localhost:4000/graphql',
-    },
-    staging: {}     // sample object for a 'staging' release channel
+  dev: {
+    apolloURI: 'localhost:4000/graphql',
+  },
+  staging: {}, // object would contain the env vars for a release channel titled 'staging'
 };
 
 /*
  * Uses Expo release channels to determine env var values.
  * As more release channels are added (ex: staging, prod), add
- * a if/else-if block to return the property in ENV corresponding to 
+ * an if/else-if block to return the property in ENV corresponding to
  * the release channel.
  */
 const getEnvVars = (env = Constants.manifest.releaseChannel) => {
-    if (__DEV__) {
-        return ENV.dev;
-    } else if (env === 'staging') {
-        return ENV.staging;
-    }
-}
+  return ENV.dev;
 
-export default getEnvVars;
\ No newline at end of file
+  /* Sample logic to determine env vars when multiple release channels are present:
+   *
+   *  if (__DEV__) {
+   *      return ENV.dev;
+   *  } else if (env === 'staging') {
+   *      return ENV.staging;
+   *  }
+   *
+   */
```

Comment:
This code block's demarcated off as sample logic -- are we sure we want this on `main`?

Reply:
perhaps it's a convenient implementation example, but just wanted to draw your attention to it.

Reply:
Good point. My goal here was to indicate that more logic is needed when new release channels are added (as I expect will be necessary at some point). But quite frankly, I don't think this looks good, so I'll just add in a link to expo's documentation for future reference.